install/kubernetes/cilium/values.yaml

```diff
@@ -2633,6 +2633,46 @@ clustermesh:
         #     cpu: 100m
         #     memory: 100Mi
 
+    kvstoremesh:
+      # -- Enable KVStoreMesh. KVStoreMesh caches the information retrieved
+      # from the remote clusters in the local etcd instance.
+      enabled: false
+
+      # -- KVStoreMesh image.
+      image:
+        override: ~
+        repository: "quay.io/cilium/kvstoremesh-ci"
```

Comment:
this appears to be a CI image, is that intended?

Reply:
I've double-checked, and also the repositories for the other components point to CI images. It seems that they are sourced from [here](https://github.com/giorio94/cilium/blob/mio%2Fkvstoremesh-helm/install/kubernetes/Makefile.values#L12-L29) (and RELEASE is not set by default).